src/scss/index.scss

```diff
@@ -103,5 +103,20 @@ $color-selection-bg           : inherit;
 // <hr> border color
 $hr-color                     : $grey--lightest;
 
+// =================================
+// Wallpapers 
+// ---
+// Wallpaper colours are used for the background colour for 
+// when an elm has overlaying a icon wallpaper
+// =================================
+
+$wallpaper-blue               : #2f7de1;
```

Comment:
Would be good to go through these with whoever worked on them to see if we can combine these with colours already in use.

So it looks like:

- The red is extremely similar to `$brand--red`.  We ideally shouldn't be using two colours so close to one another to keep the palette slimmer.
- The red--light/orange looks like it's the one from the colour rays?  If so, this was adjusted to be more AA accessible for web in the last 'slingshot' changes, but should likely add this as a $brand-rays--orange` variable and use that.
- Same for the blue, looks like the rays blue?  If so, should add this as `$brand-rays--blue`
- Same for the green – should add as `$brand-rays--green`
- Not sure where the `green-light` comes from?  Is that from the rays (it looks like the rays equiv would be `#94d600`).